bin/mksetup

```diff
@@ -86,8 +86,10 @@ if (! $?DYLD_LIBRARY_PATH) then
 endif
 
 # setup EUPS, using EUPS
-# n.b.: note the '-f (none)' flag -- this ensures eups won't show up in 'eups list' and similar
-eval `%(setup)s "DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}" eups -r "$EUPS_DIR" -f "(none)"`
+# n.b.: we reset the flavor in SETUP_EUPS so eups doesn't show up in 'eups list'
+#       and similar. This is for backwards compatibility.
+eval `%(setup)s "DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}" eups -r "$EUPS_DIR"`
+setenv SETUP_EUPS "eups LOCAL:$EUPS_DIR -f (none) -Z (none)"
```

Comment:
This breaks spaces in paths. Commit ebffe80be534515969188dcf9eefca9190d894fc changed the way local versions are handled so that the version string is now created by `Product._encode_dir()`.